### PR TITLE
feat: zulip release notifications

### PR DESCRIPTION
**Pull request by danieldietzler**

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]

File: src/config.ts [hunk 1]
```diff
@@ -7,12 +7,16 @@ export const getConfig = () => {
...
+  const zulipApiKey = process.env.ZULIP_API_KEY;
+  const zulipDomain = process.env.ZULIP_DOMAIN;
+  const githubWebhookSlug = process.env.GITHUB_SLUG;
+  const githubStatusWebhookSlug = process.env.GITHUB_STATUS_SLUG;
   const stripeWebhookSlug = process.env.STRIPE_PAYMENT_SLUG;
   const commitSha = process.env.COMMIT_SHA;
 
-  if (!clientId || !clientSecret || !databaseUri || !botToken) {
-    console.log({ clientId, clientSecret, databaseUri, botToken });
+  if (!clientId || !clientSecret || !databaseUri || !botToken || !zulipUsername || !zulipApiKey || !zulipDomain) {
```
Comment (jrasm91) [inline src/config.ts:18]: we should clean this up in another PR - probably migrate to a class config dto or such.

File: .vscode/launch.json
Comment (jrasm91) [inline .vscode/launch.json:1]: we should change this to match immich, so just attach and start with debug port by default imo.

File: src/repositories/zulip.repository.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Logger } from '@nestjs/common';
+import { IZulipInterface, MessagePayload, type ZulipConfig } from 'src/interfaces/zulip.interface';
+// @ts-expect-error: that stupid sdk does not have types
+import zulip from 'zulip-js';
+
+export class ZulipRepository implements IZulipInterface {
+  private logger = new Logger(ZulipRepository.name);
+  private zulip: any;
```
Comment (jrasm91) [inline src/repositories/zulip.repository.ts:8]: No reason we can't give an accurate, custom type here:
```ts
type Zulip = {
  messages: {
    send: ({ to: string, type: 'stream', topic: string, content: string }) => Promise<string>
   }
}
```

File: src/config.ts (hunk 1 above)
Comment (danieldietzler) [reply src/config.ts:18]: Yeah agreed, it's kind of a mess tbh

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**